Cargo.toml

```diff
@@ -17,8 +17,11 @@ linux-native = ["dep:linux-keyutils"]
 apple-native = ["dep:security-framework"]
 windows-native = ["dep:windows-sys", "dep:byteorder"]
 
-sync-secret-service = ["dep:dbus-secret-service"]
-async-secret-service = ["dep:secret-service", "dep:zbus"]
+keyutils = ["dep:linux-keyutils"]
+secret-service = []
```

Comment:
I don't like the introduction of these "do-nothing" features.  It seems you have introduced them just to make the configuration logic look simpler, but the configuration logic is not visible to clients and these features are.